extensions/db-diagram/src/models/databaseModel.ts

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import IDatabaseModel from './IDatabaseModel';
+import ITableModel from './ITableModel';
+
+export default class DatabaseModel implements IDatabaseModel {
+
+	name: string;
+	summary: string;
+	tables: ITableModel[];
+
+
+	//constructor
+	constructor(name: string, summary: string, tables: ITableModel[]) {
```

Comment:
You can put `private readonly name: string` in the constructor and avoid doing the this.name=name, etc etc

https://www.typescriptlang.org/docs/handbook/classes.html#parameter-properties